crypto/crypto_test.go

```diff
@@ -730,3 +730,11 @@ func TestTealSign(t *testing.T) {
 	verified2 := TealVerify(pk, data, addr, sig1)
 	require.False(t, verified2)
 }
+
+func TestGetApplicationAddress(t *testing.T) {
+	appID := uint64(77)
+	expected := "PCYUFPA2ZTOYWTP43MX2MOX2OWAIAXUDNC2WFCXAGMRUZ3DYD6BWFDL5YM"
```

Comment:
how is this expected value derived?

Reply:
It's from https://github.com/algorand/go-algorand/blob/412aef5ed061d96d02edfc6fa9ead6773907e66c/data/basics/userBalance_test.go#L254